package.json

```diff
@@ -15,7 +15,11 @@
     "url": "https://github.com/polkadot-js/apps.git"
   },
   "sideEffects": false,
-  "version": "0.116.1",
+  "version": "0.118.1",
```

Comment:
Can we change the version to "0.118.1-n0"? "n can stand for Nodle as the flavour of this version and "0" after "n" means the first nodle version based on 0.118.1.

Reply:
Yeah, got it.